connectors/memory/memory_test.go

```diff
@@ -86,6 +86,27 @@ var clusteredEi = &dosa.EntityInfo{
 	},
 }
 
+var clusteredByTimeEi = &dosa.EntityInfo{
+	Ref: &testSchemaRef,
+	Def: &dosa.EntityDefinition{
+		Columns: []*dosa.ColumnDefinition{
+			{Name: "f1", Type: dosa.String},
+			{Name: "c1", Type: dosa.Int64},
+			{Name: "c2", Type: dosa.Timestamp},
+		},
+		Key: &dosa.PrimaryKey{
+			PartitionKeys: []string{"f1"},
+			ClusteringKeys: []*dosa.ClusteringKey{
+				{Name: "c1", Descending: false},
+				{Name: "c2", Descending: true},
+			},
+		},
+		Name: "t3",
+		Indexes: map[string]*dosa.IndexDefinition{
```

Comment:
Nit: Probably don't need an index for this `EntityInfo` since we never test index functionality with this entity.